pkg/security/secl/model/model.go

```diff
@@ -83,6 +83,13 @@ func (m *Model) ValidateField(field eval.Field, fieldValue eval.FieldValue) erro
 		if value := fieldValue.Value; value != -int(syscall.EPERM) && value != -int(syscall.EACCES) {
 			return errors.New("return value can only be tested against EPERM or EACCES")
 		}
+	case "bpf.map.name":
```

Comment:
Missing `fallthrough`?

Reply:
I moved it in the switch case below with a coma. Thanks !